recipes/zetcd/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set import_path = "github.com/etcd-io/zetcd" %}
+{% set version = "0.0.5" %}
+
+{% set name = import_path.split('/')[-1] %}
+{% set pkg_src = ('src/'+import_path).replace('etcd-io',
+                                              'coreos').replace("/", "\\" if win else "/") %}
+
+package:
+  name: {{ name|lower }}
+  version: v{{ version }}
+
+source:
+  - folder: {{ pkg_src }}
+    git_url: https://{{ import_path }}
+    git_tag: v{{ version }}
+
+build:
+  number: 0
+  script:
+    - pushd {{ pkg_src }}
+    - go install -v
+        -ldflags "-w -X {{pkg_src}}/version.Version={{ version }} -X {{pkg_src}}/version.SHA={{ GIT_FULL_HASH }}"
+        ./cmd/...
+
+requirements:
+  build:
+    - {{ compiler("go") }}
+
+test:
+  commands:
+      - which zetcd  # [not win]
+      - where zetcd  # [win]
+      - zkctl --help
+      - zkboom --help
+
+about:
+  home: https://github.com/etcd-io/zetcd
+
+  license: Apache 2.0
```

Normalize the license
```suggestion
  license: Apache-2.0
```